### PR TITLE
Delete test file + fix CI 

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes https://github.com/sunpy/ndcube/issues/624

### CODE REVIEW

File: ndcube/utils/cube.py [hunk 1]
```diff
@@ -252,7 +252,7 @@ def propagate_rebin_uncertainties(uncertainty, data, mask, operation, operation_
     if not propagation_operation:
         if operation in {np.sum, np.nansum, np.mean, np.nanmean}:
             propagation_operation = np.add
-        elif operation in {np.product, np.prod, np.nanprod}:
+        elif operation in {np.prod, np.nanprod}:
```
Comment (nabobalis) [inline ndcube/utils/cube.py:255]: This probably isnt ok.

File: ndcube/utils/cube.py [hunk 2]
```diff
@@ -252,7 +252,7 @@ def propagate_rebin_uncertainties(uncertainty, data, mask, operation, operation_
     if not propagation_operation:
         if operation in {np.sum, np.nansum, np.mean, np.nanmean}:
             propagation_operation = np.add
-        elif operation in {np.product, np.prod, np.nanprod}:
+        elif operation in {np.prod, np.nanprod, np.product}:
```
Comment (nabobalis) [inline ndcube/utils/cube.py:255]: product will be removed eventually but for now, move it to the end.

**Review by DanRyanIrish — APPROVED** [review]

**Comment by DanRyanIrish** [comment]
Any objection if I merge this @nabobalis ?

**Comment by nabobalis** [comment]
> Any objection if I merge this @nabobalis ?

No go ahead.

**Pull request merged**